### PR TITLE
[Enhancement]correct error message when creating materialized view in aggregation model table or update model table

### LINKED ISSUE #5707: Inaccurate error message when creating materialized view in aggregation model table or update  model table
If user wants to create materialized view with a mv column which is transformed from original column  in aggregation model table or update  model table.
StarRocks will throw a inaccurate error message "Column[mv_xxx] does not exist"

This is a confusing error message, the user does not know what wrong behavior he has done.

### Steps to reproduce the behavior (Required)
1. CREATE TABLE 
```
CREATE TABLE example_table     (     k1 TINYINT,     k2 DECIMAL(10, 2) DEFAULT "10.5",     v1 int replace,     v2 int replace     )     ENGINE=olap     AGGREGATE KEY(k1, k2)     DISTRIBUTED BY HASH(k1) BUCKETS 32     PROPERTIES ("replication_num"="1");
```
2.  create materialized view
```
create materialized view mv as select k1, k2, bitmap_union(to_bitmap(v1)) from example_table group by k1,k2;
ERROR 1064 (HY000): Unexpected exception: Column[mv_bitmap_union_v1] does not exist
```

### Expected behavior (Required)
ERROR 1064 (HY000): errCode = 2, detailMessage = The mv column of aggregation model or update model table cannot be transformed from original column[v1]
### StarRocks version (Required)
 2.1.4 d965a4f

### PR DESCRIPTION
**Pull request by zhuxt2015**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] others

## Which issues of this PR fixes ：

Fixes #5707

### CODE REVIEW

**Comment by zhuxt2015** [comment]
run ci again  please

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/alter/MaterializedViewHandler.java|2|2|100.00%|

**Review by kangkaisen — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/alter/MaterializedViewHandler.java [hunk 1]
```diff
@@ -468,7 +468,10 @@ private List<Column> checkAndPrepareMaterializedView(CreateMaterializedViewStmt
                 if (mvColumnItem.isKey()) {
                     ++numOfKeys;
                 }
-                Preconditions.checkNotNull(baseColumn, "Column[" + mvColumnName + "] does not exist");
+                Preconditions.checkNotNull(baseColumn,
+                        "The materialized view column of aggregation or unique or primary table " +
```
Comment (Astralidea) [inline fe/fe-core/src/main/java/com/starrocks/alter/MaterializedViewHandler.java:472]: nit:should here carry mvColumnName information?
The materialized view column[" + mvColumnName + "] of

**Review by Astralidea — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/alter/MaterializedViewHandler.java|2|2|100.00%|

**Review by Astralidea — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Pull request merged**